tenablesc.json

```
@@ -7,15 +7,15 @@
     "type": "network security",
     "license": "Copyright (c) 2017-2024 Splunk Inc.",
     "main_module": "tenablesc_connector.py",
-    "app_version": "2.3.1",
+    "app_version": "2.3.2",
```

Since this adds a new action, let's bump to 2.4.0
```suggestion
    "app_version": "2.4.0",
```